src/common/utility.c

```diff
@@ -568,7 +568,9 @@ double dt_util_gps_string_to_number(const gchar *input)
   gchar **list = g_strsplit(input, ",", 0);
   if(list)
   {
-    if(list[2] == NULL) // format DDD,MM.mm{N|S}
+    if(list[1] == NULL) // Already in decimal format
+     res = g_ascii_strtod(list[0], NULL) ;
```

Comment:
Indentation fix needed.

Reply:
Sorry, I swear I was trying to make things easier for you all.  Sorry, it has been such a pain.  
Fixed now.